packages/@glimmer/application/src/components/component-managers/template-only.ts

```diff
@@ -0,0 +1,173 @@
+import { Owner } from '@glimmer/di';
+import { Tag } from '@glimmer/reference';
+import {
+  ComponentManager as VMComponentManager,
+  RuntimeResolver,
+  ComponentCapabilities,
+  CapturedArguments,
+  Option,
+  Invocation,
+  Environment,
+  WithAotStaticLayout,
+  Destroyable,
+  JitRuntimeResolver,
+  AotRuntimeResolver,
+  CompilableProgram,
+  Template,
+  WithJitStaticLayout,
+  ProgramSymbolTable,
+} from '@glimmer/interfaces';
+import { PathReference, CONSTANT_TAG } from '@glimmer/reference';
+import { DEBUG } from '@glimmer/env';
+
+import { RootReference, TemplateOnlyComponentDebugReference } from '../../references';
+import {
+  ComponentDefinitionState,
+  AotComponentDefinition,
+  JitComponentDefinition,
+} from '../component-definition';
+
+export const CAPABILITIES: ComponentCapabilities = {
+  attributeHook: true,
+  createArgs: true,
+  createCaller: false,
+  createInstance: true,
+  dynamicLayout: false,
+  dynamicScope: false,
+  dynamicTag: true,
+  elementHook: true,
+  prepareArgs: false,
+  updateHook: true,
+  wrapped: false,
+};
+
+export class ComponentStateBucket {
+  public name: string;
+
+  constructor(public args: CapturedArguments) {
+    this.args = args;
```

Comment:
not needed

Reply:
Nice catch, surprised this isn't caught by tslint